selendroid-standalone/src/test/java/io/selendroid/standalone/android/impl/DefaultHardwareDeviceTests.java

```diff
@@ -61,6 +62,7 @@ private void cleanUpDevice(AndroidDevice emulator) throws Exception {
     Assert.assertFalse(installedAPKs.contains(AUT_PACKAGE));
   }
 
+  @Ignore("Fail. Requires an active emulator to install the selendroid apk.")
```

Comment:
something doesn't seem right here... this test mocks everything, it should not require an active emulator or device.